index_api-v2.yaml

```diff
@@ -0,0 +1,722 @@
+                      profile_url: "https://somenode.org/optional-subdirectory/node-profile.json"
+                      status: "deleted"
+                      last_updated: 1601979232403
+                      profile_hash: "c24d14c2c75f55d334a7e0ccf4d35a063a2582a7abb91e16d326f6613b9602bf"
+                validation_failed_no_profile_url:
+                  value:
+                    data:
+                      node_id: "a55964aeaae9625dc2b8dbdb1c4ce0ed1e658483f44cf2be1a6479fe5e144d38"
+                      profile_url: "https://somenode.org/optional-subdirectory/node-profile.json"
+                      status: "validation_failed"
+                      failure_reasons:
+                        - "Could not read from profile_url: https://somenode.org/optional-subdirectory/node-profile.json"
+                validation_failed_with_schema:
+                  value:
+                    data:
+                      node_id: "a55964aeaae9625dc2b8dbdb1c4ce0ed1e658483f44cf2be1a6479fe5e144d38"
+                      profile_url: "https://somenode.org/optional-subdirectory/node-profile.json"
+                      status: "validation_failed"
+                      failure_reasons:
+                        - "example_schema-v1.geolocation.lat: Invalid type. Expected: number, given: string"
+                validation_failed_no_schema:
+                  value:
+                    data:
+                      node_id: "a55964aeaae9625dc2b8dbdb1c4ce0ed1e658483f44cf2be1a6479fe5e144d38"
+                      profile_url: "https://somenode.org/optional-subdirectory/node-profile.json"
+                      status: "validation_failed"
+                      failure_reasons:
+                        - "Could not read from schema: example_schema-v1"
+                post_failed:
+                  value:
+                    data:
+                      node_id: "a55964aeaae9625dc2b8dbdb1c4ce0ed1e658483f44cf2be1a6479fe5e144d38"
+                      profile_url: "https://somenode.org/optional-subdirectory/node-profile.json"
+                      status: "post_failed"
+                      last_updated: 1601979232403
+                NodeNotFoundInIndex:
+                  value:
+                    message: "Could not find node_id: {node_id}"
+        429:
+          $ref: "#/components/responses/TooManyRequests"
+        500:
+          $ref: "#/components/responses/ServerError"
+    delete:
+      tags:
+        - Node Endpoints
+      summary: Delete a node from the index
+      description: |
+        A node can delete its profile from the index at any time simply by removing the profile from its `profile_url` on its website and then sending this DELETE request to the index.
+
+        The index will first confirm the profile is no longer available at the `profile_url` (node's website should return a `404 - Not Found` error) and then remove the profile from its records.
```

Comment:
See comment above re sites that don't necessarily return a 404 when they should because of redirection.

Reply:
Noted. I preemptively used the word "should" here (not "must") for a reason. ;-)